CommonsServer/Strategies.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Extensions.Configuration;

```

Comment:
Folder CommonsServer no longer holds a project, I don't think Strategies file belongs there. The whole folder should be removed.